relayer-cli/src/commands/create/connection.rs

```diff
@@ -0,0 +1,159 @@
+use abscissa_core::{Command, Options, Runnable};
+
+use ibc::ics02_client::state::ClientState;
+use ibc::ics24_host::identifier::{ChainId, ClientId};
+use ibc::Height;
+use ibc_relayer::config::StoreConfig;
+use ibc_relayer::connection::Connection;
+use ibc_relayer::foreign_client::ForeignClient;
+
+use crate::commands::cli_utils::{spawn_chain_runtime, ChainHandlePair, SpawnOptions};
+use crate::conclude::{exit_with_unrecoverable_error, Output};
+use crate::prelude::*;
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct CreateConnectionCommand {
+    #[options(
+        free,
+        required,
+        help = "identifier of the side `a` chain for the new connection"
+    )]
+    chain_a_id: ChainId,
+
+    #[options(free, help = "identifier of the side `b` chain for the new connection")]
+    chain_b_id: Option<ChainId>,
+
+    #[options(help = "delay period parameter for the new connection", no_short)]
+    delay: Option<u64>,
```

Comment:
At a higher level, I am not sure if we should add the delay flag now. It might give the impression the relayer handles properly the packet delay feature which we will not do until V3. We basically ignore the delay value when we relay packets.

Reply:
I would still keep the flag. It's one thing to create a connection with certain parameters, and another to relay packets on that connection.

When relaying, however, we should make it clear (using `warn!("..")`) that Hermes v0.1-0.2 does not obey the `delay_period`.

Reply:
f2dc88cf

Reply:
> I would still keep the flag. It's one thing to create a connection with certain parameters, and another to relay packets on that connection.

I don't understand the difference. Packets relayed over all channels on a non-zero delay connection must be delayed. Why have the flag at all if it doesn't make any difference? What would we write in the documentation? "This option when specified is ignored". I would have absolutely no problem if this PR would include the delay packet PR in same release. As it is I have some doubts.

Reply:
It's two different commands. The flag in this PR refers to the connection handshake, and this flag implies nothing about relaying packets. But good point, I can see why having this flag could be confusing (if you assume that all commands are consistent among themselves) so I removed it fd9f9d64.